less/modifiers.less

```diff
@@ -52,9 +52,9 @@ html[data-color-profile=default] {
 
 /**
  * Remove img tags with aria-hidden
- * TODO: might need to be img tags without aria-label or alt as there are
- * content images with aria-hidden that need to be shown (gmcq)
+ * Remove all background images
  */
 .a11y-no-background-images {
   img[aria-hidden=true] { display: none }
+  * { background-image: none !important; }
```

Comment:
@kirsty-hames
I didn't want find all the elements which have `background-image` on the style attribute, as that could get a bit messy. Instead you can see I've targeted the vanilla theme's `.has-bg-image` class for the filters. Should this be changed from `* { background-image: none !important; }` to `.has-bg-image { background-image: none !important; }` as well to make it as specific?

Reply:
Hey @oliverfoster, yes please. Hiding CSS background images isn't an issue, only ones set via Vanilla config so we should be specific with targeting `.has-bg-image`.